ayup/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,25 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> facts;
+
+  @Override
+  public void init() {
+    facts = new ArrayList<>();
+    facts.add("I develop Nintendo GameBoy Advance ROMs in my spare time");
+    facts.add("I was a YouTube star back in the days <a href='https://youtube.com/thefredo1000'>Check out my channel</a>");
```

Comment:
You need to do something special to the text if you want this to work.. Not exactly sure what it is, but I ran your server and it doesn't actually link, just sets the text: 
![image](https://user-images.githubusercontent.com/8700194/86177500-9e05b600-badb-11ea-9a17-94b02643a73a.png)

Reply:
Thanks for noting that, I didn´t notice.

Reply:
Done